prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -1,54 +1,87 @@
 
-In this mode, the publisher will be responsible for the following actions:
+This article guides you on using Prebid SDK with a custom ad server. We will explore how to do the following:
 
-* Call fetchDemand with extended targetingDict callback
-* Retrieve targeting keys from the extended fetchDemand function
-* Convert targeting keys into the format for your ad server
-* Pass converted keys to your ad server
-* Render ad with Prebid Universal Creative or custom renderer
+1. Initialize the Prebid Mobile Android SDK.
+2. Define first party data and set up adUnits.
+3. Initialize the ad auction.
+4. Render the results.
 
-This approach is available for the following ad formats:
+## Initializing the Prebid SDK
 
-* Display Banner via `BannerAdUnit`
-* Video Banner and Instream Video via `VideoAdUnit`
-* Display Interstitial via `InterstitialAdUnit`
-* Video Interstitial via `VideoInterstitialAdUnit`
-* Rewarded Video via `RewardedVideoAdUnit`
-* Native Styles via `NativeRequest`
+Before implementing the instructions in this guide, you need to ensure that you have correctly initialized the Prebid Mobile SDK in your app. The initialization method has the following pattern:
 
-The basic integration steps for these ad units you can find on the page for integration using [Original API](/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html). The difference is that you should use  the `fetchDemand` function with the following signature:
+```java
+Prebid.init(context, accountID, adServer, host);
+```
 
-```kotlin
+To use the above code snippet, you will need to [setup a Prebid server.](/prebid-mobile/prebid-mobile-getting-started.html#set-up-prebid-server) You can either register with a [Prebid.org member that hosts Prebid Server](https://prebid.org/managed-services/) or [setup your own Prebid server.](/prebid-server/hosting/pbs-hosting.html)
+
+* `context`: This parameter represents the Context of the Android application. It is typically obtained using `getApplicationContext()` or `this` (if called from within an Activity). The Context is essential for the SDK to interact with the Android environment.
+
+* `adUnitId`: This is the value of the adunit name in the ad server. e.g. for GAM, it's the adunit name. Other ad servers may call this something different, but the idea is that the buyers want to know which slot in the app is up for auction for targeting and reporting. An example would be "/home/upper-mobile-banner".
+
+* `accountId`: The accountId is used to define the accountID and the "account settings" used for this app in Prebid Server. These allow your Prebid Server to lookup account details and attributes like timeout and price granularity.
+
+* `adServer`: This parameter specifies the ad server you are using. It is an enum value of the type `Prebid.AdServer`. The possible values include `Prebid.AdServer.DFP` for Google Ad Manager and `Prebid.AdServer.MOPub` for MoPub.
```

Comment:
This whole page is about "custom" integrations. Why would they need to set DFP or MoPub? And isn't MoPub long dead?